src/SourceBuild/content/eng/pipelines/templates/stages/vmr-scan.yml

```diff
@@ -45,6 +46,7 @@ stages:
         ./.dotnet/dotnet darc vmr scan-cloaked-files
         --vmr "$(Build.SourcesDirectory)"
         --tmp "$(Agent.TempDirectory)"
+        --baseline-file "(Build.SourcesDirectory)/src/installer/src/VirtualMonoRepo/baseline-file.txt"
```

Comment:
Why is a baseline file necessary in this scenario?

Reply:
Oh yeah, good catch. It shouldn't be used here

Reply:
yup, I noticed that too. I'll need to fix that in the code, since it's a required parameter currently